views/media/field-row.tt

```diff
@@ -0,0 +1,32 @@
+[% DEFAULT
+    labelSize = 2
+    fieldSize = 12 - labelSize
+    type      = "text"
+%]
+<div class="row">
+    <div class="large-[% labelSize %] columns">
+        <label for="[% name %]">[% label %]</label>
+    </div>
+    <div class="large-[% fieldSize %] columns">
```

Comment:
To clarify, when I say "rather odd" I mean there's a lot of separation.  Or... maybe my browser was hanging onto a cached version of the CSS file, because when I refreshed a lot of the odd spacing disappeared.  But the Delete checkbox and its associated label are still separated by the entire width of the page, making both elements look orphaned.